reactioncommerce/reaction
# Order filtering by date

Resolves #2359 

### How to test
- Make a couple of orders (better if you change the time orders are created to observe that the date filter works)
- Pick a range of dates
- Observe that orders are filtered correctly
- Try filtering by status and searching for a particular group of orders
- Observe that the date filter works in tandem with the other filter and search
